api_fhir/converters/locationConverter.py

```diff
@@ -0,0 +1,153 @@
+from location.models import HealthFacility
+
+from api_fhir.apiFhirConfiguration import ApiFhirConfiguration
+from api_fhir.converters import BaseFHIRConverter
+from api_fhir.models import Location, ContactPointSystem, ContactPointUse
+from api_fhir.models.address import AddressUse, AddressType
+from api_fhir.models.imisModelEnums import ImisHfLevel
+import core
+
+class LocationConverter(BaseFHIRConverter):
+    @classmethod
+    def to_fhir_obj(cls, imis_hf):
+        fhir_location = Location()
+        cls.build_fhir_location_identifier(fhir_location, imis_hf)
+        cls.build_fhir_location_name(fhir_location, imis_hf)
+        cls.build_fhir_location_type(fhir_location, imis_hf)
+        cls.build_fhir_location_address(fhir_location, imis_hf)
+        cls.build_fhir_location_telcome(fhir_location, imis_hf)
+        return fhir_location
+
+    @classmethod
+    def to_imis_obj(cls, fhir_location, audit_user_id):
+        errors = []
+        imis_hf = cls.createDefaultInsuree(audit_user_id)
+        cls.build_imis_hf_identiftier(imis_hf, fhir_location, errors)
+        cls.build_imis_hf_name(imis_hf, fhir_location, errors)
+        cls.build_imis_hf_level(imis_hf, fhir_location, errors)
+        cls.build_imis_hf_address(imis_hf, fhir_location)
+        cls.build_imis_hf_contacts(imis_hf, fhir_location)
+        cls.check_errors(errors)
+        return imis_hf
+
+    @classmethod
+    def createDefaultInsuree(cls, audit_user_id):
+        imis_hf = HealthFacility()
+        # TODO legalForm isn't covered because that value is missing in the model (value need to be nullable in DB)
+        # TODO LocationId isn't covered because that value is missing in the model (value need to be nullable in DB)
+        imis_hf.offline = ApiFhirConfiguration.get_default_value_of_location_offline_attribute()
+        imis_hf.care_type = ApiFhirConfiguration.get_default_value_of_location_care_type()
+        imis_hf.validity_from = core.datetime.datetime.now()
```

Comment:
Used now() method is provided by the OpenIMIS core module which handle different types of date (Gregorian and Nepal) so the used timezone should be always the same. However I moved all operations on the date/time to the separate utils class. I believe it will improve the readability. Thanks.